src/plugins/intel_gpu/include/intel_gpu/graph/serialization/binary_buffer.hpp

```diff
@@ -38,9 +37,8 @@ class BinaryInputBuffer : public InputBuffer<BinaryInputBuffer> {
 
     void read(void* const data, std::streamsize size) {
         auto const read_size = stream.rdbuf()->sgetn(reinterpret_cast<char*>(data), size);
-        if (read_size != size) {
-            throw std::runtime_error("Failed to read " + std::to_string(size) + " bytes from stream! Read " + std::to_string(read_size));
-        }
+        OPENVINO_ASSERT(read_size == size,
+            "Failed to read " + std::to_string(size) + " bytes from stream! Read " + std::to_string(read_size));
```

Comment:
Add [GPU] tag here and above

Reply:
I updated it as you reviewed. But I don't quite understand what you mean by creating a new header file. If the current implementation is not the direction you want, please provide me a little more detailed explanation. Thank you.